main.py

```diff
@@ -1,15 +1,23 @@
 from libs.sudocode import get_code
 from libs.cleaner import code_cleaner, code_execute
+import sys
 
-if(__name__=='__main__'):
+if len(sys.argv) is 2:
```

Comment:
Everything seems perfect. Just one thing though. Would be great if you enclose your if and elif statements with parenthesis. Like 
`if(<condition>):`

Reply:
done. I added parenthesis in if and elif statements